src/extension/extension_installer.cpp

```diff
@@ -52,6 +55,9 @@ void ExtensionInstaller::installExtension() {
         extension::ExtensionUtils::getLocalExtensionDir(&context, info.name);
     if (!vfs->fileOrPathExists(localDirForExtension)) {
```

Comment:
Check for the file instead of just the directory, say if a user manually deletes the extension but not the directory.